package.json

```diff
@@ -17,6 +17,7 @@
   "dependencies": {
     "@wordpress/a11y": "0.1.0-beta.1",
     "@wordpress/hooks": "1.0.1",
+    "@wordpress/jest-console": "1.0.0",
```

Comment:
I still think this should be just `@wordpress/jest` but not very important right now

Reply:
I know, I know :) We can discuss it further when we have more functionality to add there.

Reply:
For general usefulness, might having single-purpose modules here be preferred? vs. a monolithic utility library.

I don't feel strongly one way or the other, and it likely varies by the type of module (e.g. for unit tests, I'm more likely to pick and choose very specific utility support to add).

Reply:
My initial idea was to consolidate all very specific utilities under WordPress/scripts.